src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -19,6 +20,18 @@ import { OrganizationFilters } from './OrganizationFilters';
 import { PublisherFilter } from './PublisherFilter';
 import { SeriesFilter } from './SeriesFilter';
 
+const StyledDivider = styled(Divider)(({ theme }) => ({
+  backgroundColor: theme.palette.primary.main,
+}));
+
+const StyledFilterContainer = styled(Box)(() => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.25rem',
+}));
```

Comment:
Her initieres en overflødig funksjon
```suggestion
const StyledFilterContainer = styled(Box)({
  display: 'flex',
  flexDirection: 'column',
  gap: '0.25rem',
});
```

Reply:
Fikset ved å trekke ut i egen variabel

Reply:
Da syns jeg det blir litt i klinjs med Styled over der igjen da, ved at samme problem løses på ulike måter. Personling liker jeg Styled best siden man da ikke må legge til samme sx-prop alle plasser, men om alt bare blir `Sx`-variabler er for så vidt like greit da eventuelt

Reply:
Da er det `styled` over hele fjøla